docs/get-details/dapps/smart-contracts/frontend/smartsigs.md

```diff
@@ -3,12 +3,8 @@ title: Interact with smart signatures
 This guide covers using smart signatures with the Algorand SDKs. Smart signatures are also referred to as stateless smart contracts. Smart signatures can be used to create contract accounts or to handle account delegation which is described in the [Modes](../smartsigs/modes.md) documentation. In either case, the contracts are written in [Transaction Execution Approval Language (TEAL)](../../avm/teal/index.md) or with Python using the [PyTeal](../../pyteal/index.md) library.
 
 
-!!! info
-    The example code snippets are provided throughout this page and are abbreviated for conciseness and clarity. Full running code examples for each SDK are available within the GitHub repo at [/examples/smart_contracts](https://github.com/algorand/docs/tree/master/examples/smart_contracts) and for [download](https://github.com/algorand/docs/blob/master/examples/smart_contracts/smart_contracts.zip?raw=true) (.zip).
-
-
 # Compiling TEAL program from SDKs
-Before a TEAL program can be used, it must be compiled. SDKs provide this capability. The examples in this section read a file called `sample.teal` which contains one line of TEAL code, `int 0` . This will always return `false`. So, any transactions that use this TEAL file will fail. 
+Before a TEAL program can be used, it must be compiled. SDKs provide this capability. The examples in this section use simple contract which contains one line of TEAL code, `int 1` . This will always return `true`. So, any transactions that use this TEAL file will succeed. Never use this TEAL in a production application as it approves all transactions. 
```

Comment:
```suggestion
Before a TEAL program can be used, it must be compiled. SDKs provide this capability. The examples in this section use a simple contract which contains one line of TEAL code, `int 1` . This will always return an approval for the transaction. So, any transactions that use this TEAL file will succeed. Never use this TEAL in a production application as it approves all transactions. 
```